### PR TITLE
Add as2021

### PR DESCRIPTION
**Pull request by mbaudin47**
Added content provided by @adutfoy

### CODE REVIEW

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 1]
```diff
@@ -93,12 +93,12 @@
 We want to compare  \alert{$\Var{Y^*_i}$} to \alert{$\Var{Y}$}:
  \begin{enumerate}
   \item in the case of independent variables: \alert{Sobol indices},
-  \item in the case of dependent variables: importance factors (\alert{Taylor decomposition variance}), \alert{ANCOVA indices}.
+  \item in the case of dependent variables: importance factors (\alert{Tayor decomposition variance}), \alert{ANCOVA indices}.
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: bad fix

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 2]
```diff
@@ -207,7 +207,7 @@ \subsection{Sobol Indices}
          \label{phi}
          \vect{U} = (F_1(X_1), \dots, F_n(X_n))^t) = \phi^{-1}(\vect{X})
        \end{equation}
-       then $\vect{U}$ has uniform marginals and its copula is the same as $\vect{X}$, then {\bf we can use the Hoeffding decomposition on $f \circ \phi$}.
+       then $\vect{U}$ has uniform marges and its copula is the same as $\vect{X}$, then {\bf we can use the Hoeffding decomposition on $f \circ \phi$}.
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: here too

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 3]
```diff
@@ -218,7 +218,7 @@ \subsection{Sobol Indices}
          \label{varCond}
          \Expect{Y|\vect{U}} = \Expect{Y|\vect{X}}
        \end{equation}
-       As a matter of fact : $\Expect{Y|\vect{U}} = \Expect{Y|\psi(\vect{X})}$ is the orthogonal projection in a $L_2$ sense of $Y$ on the space generated by $\psi(\vect{X})$, which is the same as the one generated by $\vect{X}$, thus the equality of the random variables  (\ref{varCond}).\\
+       As a matter of fact : $\Expect{Y|\vect{U}} = \Expect{Y|\psi(\vect{X})}$ is the orthogonal projection in a $L_2$ sens of $Y$ on the space generated by $\psi(\vect{X})$, which is the same as the one generated by $\vect{X}$, thus the equality of the random variables  (\ref{varCond}).\\
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: here too

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 4]
```diff
@@ -303,7 +303,7 @@ \subsection{An example}
 
 \begin{block}{How to proceed?}
   \begin{itemize}
-  \item We tested the independence hypothesis of the input using the Spearman coefficients: we can't reject the hypothesis with a level $95\%$
+  \item We tested the independence hypothesis of the input using the Spearmann coefficients: we can't reject the hypothesis with a level $95\%$
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: here also --> Spearman

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 5]
```diff
@@ -347,17 +347,17 @@ \subsection{Particular cases: historical measures}
...
   \item SRC, SRRC indices
-  \item Pearson, Spearman, PCC, PRCC indices
+  \item Pearson, Spearmann, PCC, PRCC indices
   \item importance factors from the \emph{Taylor decomposition}
  \end{itemize}
 \alert{When the components $X_i$ are independent, these indices are exactly particular cases of Sobol indices.}
 
  \begin{block}{If the model $f$ is linear  w.r.t. the  $X_i$ : SRC}
-  If $\alert{Y = \alpha_0 + \sum_i \alpha_i X_i}$, with \alert{ $X_i$ independent}, then we define the  \alert{ Standard Regression Coefficient (SRC)} :
+  If $\alert{Y = \alpha_0 + \sum_i \alpha_i X_i}$, whith \alert{ $X_i$ independent}, then we define the  \alert{ Standard Regression Coefficient (SRC)} :
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: here too

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 6]
```diff
@@ -347,17 +347,17 @@ \subsection{Particular cases: historical measures}
...
     \begin{equation}
-      SRC_i = \displaystyle \frac{\alpha_i^2 \Var{X_i}}{\Var{Y}}
+      SRC(X_i) = \displaystyle \frac{\alpha_i^2 \Var{X_i}}{\Var{Y}}
     \end{equation}
-    Then {\bf $SRC $ is the Sobol index} of order 1 of $X_i$ : $\boldsymbol{    SRC (Y / X_i) = S_i(Y / X_i)}$.
+    Then {\bf $SRC $ is the Sobol indice} of order 1 of $X_i$ : $\boldsymbol{    SRC (Y / X_i) = S(Y / X_i)}$.
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: here also

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 7]
```diff
@@ -393,10 +393,10 @@ \subsection{Particular cases: historical measures}
            }
        then we define the  \alert{ Standard Rank Regression Coefficient (SRRC)} :
           $$
-     SRRC(Y/X_i) = SRC(Z / U_i) = \dfrac{\alpha_i^2 \Var{U_i}}{\Var{Z}} = S_i(Z/U_i)
+     SRRC(Y/X_i) = SRC(Z / U_i) = \dfrac{\alpha_i^2 \Var{U_i}}{\Var{Z}} = S(Z/U_i)
     $$    
        
-    Then {\bf SRRC is a Sobol index} of order 1 calculated on the ranks of $X_i$ and $Y$.
+    Then {\bf SRRC is a Sobol indice} of order 1 calculated on the ranks of $X_i$ and $Y$.
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: here also

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 8]
```diff
@@ -424,8 +424,8 @@ \subsection{Particular cases: historical measures}
   Importance factors from the \emph{Taylor decomposition} have been defined in metrology first where:
   \begin{itemize}
   \item $Y=f(\vect{X})$
-   \item $\vect{X}$ is a gaussian vector with independent components with {\bf low variation coefficient} ($\sigma / \mu \ll 1$)
-   \item[$\Longrightarrow$] $f$ is linearized at $\Expect{\vect{X}}$
+   \item $\vect{X}$ is a gaussian vector with independent components whith {\bf low variation coefficient} ($\sigma / \mu \ll 1$)
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: here too

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 9]
```diff
@@ -638,7 +638,7 @@ \subsection{Indices based on the Csiszar divergence}
 \begin{frame}
   \frametitle{Csiszar Divergence}
 \small
-The sensitivity index writes:
+The sensitivity indice writes:
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: sensitivity index

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 10]
```diff
@@ -648,14 +648,14 @@ \subsection{Indices based on the Csiszar divergence}
...
-The characterization of the range of the indices is the main result for the dependence analysis.
+The characterisation of the range of the indices is the main result for the dependence analysis.
   \end{block}
 
   \begin{block}{Methodology and numeric issues}
    Works are in progress on the following challenges: 
     \begin{itemize}
-    \item \alert{how to interpret the value of the indice?} $S_i^f=0.8$: if Sobol index, it means that $80\%$ of the variance of $Y$ is explained by the variance of $X_i$... but what if Csiszar divergence?
-    \item \alert{which  $f$ to consider?} If $S_i^{f} > S_j^{f}$, do we still have $S_i^{g} > S_j^{g}$? Answer: no... thus, the hierarchization depends on $f$. We have to\emph{adapt $f$} to the needs of the study. For example, if $c(x_i, y)$ is low in some particular zones, we take a $f$ which increases the gaps to 1 in that zone. We need to build a know-how!
+    \item \alert{how to interpret the value of the indice?} $S_i^f=0.8$: if Sobol indice, it means that $80\%$ of the variance of $Y$ is explained by the variance of $X_i$... but what if Csiszar divergence?
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: once again

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 11]
```diff
@@ -673,7 +673,7 @@ \subsection{Indices based on the Csiszar divergence}
    How to proceed: 
    \begin{enumerate}
     \item On the sample $k$ of $(x_i, y)$ of size $n$, generated under the independence assumption between $x_i$ and $y$, we build the copula density $\hat{c}_k(x_i,y)$  of $(x_i,y)$ thanks to the Bernstein copula;
-    \item We repeat Step 1  $N$ times: we draw, at any point $(x_i,y)$, the percentile $5\%$ and $95\%$ of the values of $\hat{c}_k(x_i,y), 1\leq k \leq N$;
+    \item We repeat Step 1  $N$ times: we draw, at any point $(x_i,y)$, the quantile $5\%$ and $95\%$ of the values of $\hat{c}_k(x_i,y), 1\leq k \leq N$;
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: quantiles

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex [hunk 12]
```diff
@@ -64,11 +64,11 @@
 \end{itemize}
 
   \begin{block}{Why sensitivity analyses?}
-  The main objectives of sensitivity analyses are the following:
+  The main objectives of sensitivity analyses may be:
 \begin{enumerate}
  \item \alert{remove some variables} which are not influential on the feature of interest, within a context of high dimension,
- \item\alert{ prioritize variables } in order to prioritize modeling efforts: we need a  \emph{relative} quantification: we want that if $S_i < S_j$ then $\Prob{\hat{S}_j \leq \hat{S}_i} \leq \varepsilon$
- \item \alert{quantify the impact of a variable}: we need a \emph{exact} quantification: we want that $\Prob{|\hat{S}_i - S_i| > \eta} \leq \varepsilon$
+ \item\alert{ prioritize variables } in order to prioritize modeling efforts: we need a  \emph{relative} quantification
```
Comment (sofianehaddad) [inline 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: I would say "Rank variables n order to priotirize.."

**Comment by mbaudin47** [comment]
Thank your very much Sofiane. I fixed as many issues as I could in the update commit.

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 1 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed.

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 2 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed.

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 3 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: I cannot understand this: it is as if a Gremlins were to insert new bugs! Fixed.

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 4 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 5 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 8 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 9 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 10 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed.

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 11 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed.

File: 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex (hunk 12 above)
Comment (mbaudin47) [reply 2021/Courses/Sensitivity/AnaSensibilite_PRACE_english.tex]: Fixed, and rephrased.

**Comment by mbaudin47** [comment]
As a side note, 'Indices' is the plural form of the noun 'index'.

**Pull request merged**